Update n-tier-content.md (#4465)
MicrosoftDocs/architecture-center: docs/guide/architecture-styles/n-tier-content.md

Layers are a way to separate responsibilities and manage dependencies. Each layer has a specific responsibility. A higher layer can use services in a lower layer, but not the other way around.

> Tiers are physically separated, running on separate machines and can be either strict or relaxed: in the strict way, a request must go through adjacent tiers, one by one, and can't skip any tier in between. For example, from WAF to Web tier then to Middle Tier 1. In contrast, in the relaxed approach, the request can skip some tiers if it's necessary. The strict approach has more latency and overhead, and the relaxed approach has more couplings and subsequently it's more difficult to change. A system can use a hybrid approach: having both relaxed and strict tiers where necessary.

Reviewer: ```suggestion
Tiers are physically separated, running on separate machines. Contractually, the tier can have their communication models be strict or relaxed. In the strict model, a request must go through adjacent tiers, one by one, and can't skip any tier in between. For example, from the web application firewall to the web tier, then to middle tier 1, and so on. In contrast, in the relaxed approach, the request may skip some tiers if it's necessary. The strict approach has more latency and overhead, and the relaxed approach has more couplings and subsequently it's more difficult to change. A system can use a hybrid approach: having both relaxed and strict tiers where necessary.
```

- Portability between cloud and on-premises, and between cloud platforms.
- Less learning curve for most developers.

> - Relatively low cost

Reviewer: ```suggestion
- Relatively low cost by not rearchitecting the solution
```

- Monolithic design prevents independent deployment of features.
- Managing an IaaS application is more work than an application that uses only managed services.
- It can be difficult to manage network security in a large system.

> - Workflows typically span across multiple tiers

Reviewer: ```suggestion
- User and data flows typically span across multiple tiers, adding complexity to concerns like testing and observability.
```
